finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,93 @@
+//   fullEnergy should add 150 to your current energy √
+var fullPower = 500 * power;
+var fullEnergy = 150 + energy;
+
+// Declare two variables - isHuman and identityConcealed - assigned to booleans √
+var isHuman = true;
+var identityConcealed = false;
+
+// Declare two variables - archEnemies AND sidekicks √
+//   archEnemies should be an array of at least 3 different enemy strings √
+//   sidekicks should be an array of at least 3 different sidekick strings √
+var archEnemy = ["Jocker", "Two-Face", "Riddler"];
+var sidekicks =["Robin", "Catwoman", "Alfred"] ;
+
+// Print the first sidekick to your console √
+console.log(sidekicks[0]);
+
+// Print the last archEnemy to the console √
+console.log(archEnemy[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemy.push("Penguin");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemy);
 // Remove the first sidekick from the sidekicks array
-
+sidekicks.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
-// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
-//   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
-//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse √
+//   - dangerLevel should be an integer √
+//   - saveTheDay should be a string a hero would say once they save the day √
+//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel √
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
+  console.log(dangerLevel,saveTheDay,badExcuse)
+}
+assessSituation(7, "You are safe now!", "I dont know if is we can do this");
 
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
+  console.log(dangerLevel, saveTheDay, badExcuse);
+}
+assessSituation(7 ,"You are safe now!", "I dont know if is we can do it");
+
+if (assessSituation[0] > 50){
+  console.log(assessSituation[2]);
+} else if (assessSituation[0]>= 10){
+  console.log(assessSituation[1]);
+} else (assessSituation[0]< 10);{
+  console.log("Meh. Hard pass.");
+}
 
 //Test Cases
-var announcement = 'Never fear, the Courageous Curly Bracket is here!'
```

Comment:
You don't need to delete the Test cases - these are there for you to test your function that you defined above.